src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/CompletionIntegrationTests.cs

```diff
@@ -125,15 +125,15 @@ public async Task Snippets_DoNotTrigger_OnDelete()
 
             <PageTitle>Test</PageTitle>
 
-            <div></div>
+            <div>Hello</div>
             """,
             open: true,
             ControlledHangMitigatingCancellationToken);
 
         var textView = await TestServices.Editor.GetActiveTextViewAsync(HangMitigatingCancellationToken);
         await TestServices.Editor.WaitForComponentClassificationAsync(ControlledHangMitigatingCancellationToken);
 
-        await TestServices.Editor.PlaceCaretAsync("<di", charsOffset: 1, ControlledHangMitigatingCancellationToken);
+        await TestServices.Editor.PlaceCaretAsync("Hel", charsOffset: 1, ControlledHangMitigatingCancellationToken);
```

Comment:
@ryzngard I had to update this test because it started failing. This change seems to still match the original intention of the fix (https://github.com/dotnet/razor/pull/9443), because we're not showing _snippets_ in completion on deletion, but the test was failing because it was still seeing Html element completion.

I'm not sure why the behaviour changes with this PR though.

Reply:
This is definitely odd... Let's talk offline so I can understand a bit better. I don't see it as an issue as long as manually testing when a user presses delete we don't add snippets to the completion list.

Reply:
I confirmed that manually, and the screenshots from the failed test run show it too. Only tags, no snippets (eg, no "iframe")
![09 19 52-CompletionIntegrationTests Snippets_DoNotTrigger_OnDelete-NullException](https://github.com/dotnet/razor/assets/754264/b1d04e72-0f66-49ed-a401-447a1e64fa6e)